gallery.html

```diff
@@ -0,0 +1,93 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="gallery.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+Garamond:wght@400;700&family=Ibarra+Real+Nova:ital@1&family=Lato&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant+SC:wght@400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <title>Artă nouă și veche</title>
+  </head>
+  <body>
+    <header class="header-bar">
+      <a href="#" class="logo-link"><img src="logo/Artă nouă și veche-logo.png" alt="logo Artă nouă și veche" class="logo-img"/></a>
+      <nav class="nav-bar small">
+        <ul class="line-icons">
+          <li class="list"><a href="index.html" class="menu-btn home"><span class="material-symbols-outlined home-icon">
+            home
+            </span></path></svg></a></li>
+          <li class="list"><a href="about.html" class="menu-btn about"><span class="material-symbols-outlined about-icon">
+            groups
+            </span></a></li>
+          <li class="list"><a href="gallery.html" class="menu-btn gallery"><span class="material-symbols-outlined gallery-icon">
+            image
+            </span>
+          </svg></a></li>
+          <li class="list"><a href="#" class="menu-btn contact"><span class="material-symbols-outlined contact-icon">
+            person_pin_circle
+            </span></a></li>
+        </ul>
+      </nav>
+      <nav class="nav-bar big">
+        <ul class="line-icons">
+          <li class="list"><a href="index.html" class="menu-btn home">Acasă</a></li>
+          <li class="list"><a href="about.html" class="menu-btn about">Despre noi</a></li>
+          <li class="list"><a href="gallery.html" class="menu-btn gallery">Galerie</a></li>
+          <li class="list"><a href="#" class="menu-btn contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1 class="subtitle gallery-title">Galerie</h1>
+      <div class="gallery-pattern">
+        <img src="images/statuie.jpg" alt="statuie barbat" class="img-gallery img-1"/>
+        <h3 class="img-title-gallery title-1">ceva despre statuie</h3>
+        <p class="text-gallery text-1">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque a neque nobis veniam mollitia, voluptatum, aliquam, illum similique vero eaque amet.</p>
```

Comment:
Apreciez foarte mult că ai încercat să faci cât mai puțin markup (HTML), dar aici ai putea pune fiecare rând într-un element separat:

```html
      <article class="gallery">
        <img src="images/statuie.jpg" alt="statuie barbat" class="gallery__img" />
        <h3 class="gallery__title">ceva despre statuie</h3>
        <p class="gallery__text">
          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque a neque nobis veniam mollitia, voluptatum,
          aliquam, illum similique vero eaque amet.
        </p>
      </article>
```

Astfel, vei avea mai puțin CSS.
Dacă vrei și mai puțin CSS, am făcut ceva asemănător cu ce încerci (imagine care alternează pe fiecare rând) la exercițiile astea: https://codesandbox.io/s/exercitiu-sub-menu-fog3yl?file=/style.css

```css
.post img {
  grid-column: 3 / 4;
  grid-row: 1 / 3;
}

.post:nth-child(2n) img {
  grid-column: 1 / 2;
}
```

La fiecare al 2-lea post (2n), pun imaginea pe prima coloană (în stânga) în loc de a treia (în dreapta). Eu acolo aveam 3 coloane, dar sper că înțelegi.